snapcraft/tests/unit/test_elf.py

```diff
@@ -379,7 +401,8 @@ def test_patch_fails_raises_patcherror_exception(self, check_call_mock):
                       'dynamically linked, interpreter '
                       '/lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32')
         elf_file = elf.ElfFile(path='/fake-elf', magic=stub_magic)
-        elf_patcher = elf.Patcher(dynamic_linker='/lib/fake-ld')
+        elf_patcher = elf.Patcher(dynamic_linker='/lib/fake-ld',
+                                  base_path='/random')
```

Comment:
Is this intentionally not `self.prime_dir`? If so, can you add a comment to the test to explain why?

Reply:
easy fix, but it is irrelevant as there are no files in there.